File: Library/Library.Model/Model/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Library.Model.Model
{
    public class Book : BaseEntity
    {
        public string Name { set; get; }

        public string Author { set; get; }

        public DateTime PublishDate { set; get; }
    }
}

File: Library.Spa/Scripts/app/book.js
(function (libApp) {
    function book() {
        var self = this;

        self.guid = ko.observable("");

        self.Name = ko.observable("");

        self.Author = ko.observable("");

        self.Date = ko.observable("");
    }
    libApp.model.Book = book;
}(window.LibApp));
